src/components/Missions/MissionsPage.js

```diff
@@ -0,0 +1,20 @@
+const MissionsItem = (prop) => {
```

Comment:
Try to name the component the same as the file.js  in this case the component is MissionItem and the file name is MissionsPage, this will save you a lot of trouble in bigger projects ;)

Reply:
Done!